### ISSUE TITLE
Getting started and LuaFileSystem

### ISSUE THREAD

**Issue by caldwecr** [Olivine-Labs/busted#550]
I read another issue where a user was getting an error similar to mine

```
caldwecr@caldwecr-dell:~/code/lua$ ~/.luarocks/bin/busted 
/usr/bin/lua5.1: /home/caldwecr/.luarocks/share/lua/5.1/pl/path.lua:28: pl.path requires LuaFileSystem
stack traceback:
	[C]: in function 'error'
	/home/caldwecr/.luarocks/share/lua/5.1/pl/path.lua:28: in main chunk
	[C]: in function 'require'
	...e/caldwecr/.luarocks/share/lua/5.1/busted/runner.lua:3: in main chunk
	[C]: in function 'require'
	...ocks/lib/luarocks/rocks/busted/2.0.rc12-1/bin/busted:3: in main chunk
	[C]: ?
```

In the other case it seemed that the user was trying to use both lua5.1 and lua5.2

I'm only trying to run lua5.1; it's the first lua I've ever used and I'm really struggling to get these tests to run.

I've tried installing the busted rock with sudo and alternatively with --local; I've done the same when trying explicitly installing LuaFileSystem.

All of the configuration _looks_ correct, but I'm sure I'm doing something terribly stupid wrong.

Here's the file I'm trying to require

```lua
-- defines a factorial function
function fact (n)
   if n == 0 then
      return 1
   else
      return n * fact(n-1)
   end
end
```

and here's the test

```lua
local factorial = require "factorial"

describe("a test", function()
	    assert.is_true(true)
)
```

I realize that assert.is_true(true) doesn't depend on the required file but obviously I have to tell the test which modules to include.

Please help; sorry I'm so new to this language.

**Comment by Tieske** [comment]
seems you need to setup your paths.
I'd guess you got LuaRocks installed?

then make sure that the LuaRocks `bin` directory is in your system path (then you should be able to call `busted` without specifying the path)

Once that is done, make sure the Lua paths are setup properly, as Busted depends on the Penlight library, which depends on `lfs` (luafilesystem). And it appears Lua cannot find the latter.

for Lua paths info see http://www.thijsschreijer.nl/blog/?p=1025

**Comment by caldwecr** [comment]
Thanks. I'm giving it a shot.

**Comment by caldwecr** [comment]
FYI I never could get this to work locally. I just created a docker container and -v the source and spec files based on https://hub.docker.com/r/jagregory/busted/builds/b5ny3thaamamprt9zswwwkb/

That worked perfectly the first time
